Update staff-recruiting/README.md (#340)
mattermost/mattermost-handbook: contributors/join-us/staff-recruiting/README.md

> ## Role Description 

Reviewer: ```suggestion
## Role description 
```

> Once you have confirmation that your new headcount has been approved, you’re ready to begin the recruitment process. Complete the [Job Description portion](https://docs.google.com/document/d/1rpTI2NKu4H_781vpx_eD9Fku7go4gGDs1P7bJAMQvAU/edit#) and share with your Recruiter, who will schedule a [Recruiter In-Take Session](https://handbook.mattermost.com/contributors/join-us/staff-recruiting/Recruiting Cadences) to review the remainder of the in-take template. 

Reviewer: ```suggestion
Once you have confirmation that your new headcount has been approved, you’re ready to begin the recruitment process. Complete the [Job Description portion](https://docs.google.com/document/d/1rpTI2NKu4H_781vpx_eD9Fku7go4gGDs1P7bJAMQvAU/edit#) and share with your Recruiter, who will schedule a [Recruiter Intake Session](https://handbook.mattermost.com/contributors/join-us/staff-recruiting/recruiting-cadences.html) to review the remainder of the intake template. 
```

> Your Recruiter will be based on the role and team. This may differ depending on Recruiting Team bandwidth, and will be communicated on a case-by-case basis. 

Reviewer: ```suggestion
Your Recruiter is assigned based on the role and team. This may differ depending on Recruiting Team bandwidth, and will be communicated on a case-by-case basis.
```

> * Technical Individual Contributor Roles -- Teresa Novoa 

Reviewer: ```suggestion
* Technical individual contributor roles - Teresa Novoa
```

> * All others -- Natalie Jew 

Reviewer: ```suggestion
* All others - Natalie Jew
```

> ### Role Leveling 

Reviewer: ```suggestion
### Role leveling 
```

> You can view the current [v1 Job Levels here](https://docs.google.com/spreadsheets/d/1kNsl7JUWDQ8MhWonys9dR3osrxEWYkKVIoECfFKSQzw/edit#gid=0). Note: HR is also working with MLT to build v2 which will help provide additional clarity. If you are unsure about how to use or assign a level to a new job req, your recruiter can help you walk through this as part of the Recruiter In-Take. 

Reviewer: ```suggestion
You can view the current [v1 Job Levels here](https://docs.google.com/spreadsheets/d/1kNsl7JUWDQ8MhWonys9dR3osrxEWYkKVIoECfFKSQzw/edit#gid=0). 
**Note:** HR is also working with MLT to build v2 which will help provide additional clarity. If you are unsure about how to use or assign a level to a new job requisition, your Recruiter can help you walk through this as part of the Recruiter Intake. 
```

* Feedback on the specific skill/focus area assigned with examples to help contextualize your rating  

> * Potential [blindspots](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check) or [brown m&ms](https://handbook.mattermost.com/company/about-mattermost/mindsets#brown-m-and-ms) that may require either a future interviewer to ask about or the Hiring Manager to cover during a reference check.

Reviewer: ```suggestion
* Potential [blindspots](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check) or [brown M&Ms](https://handbook.mattermost.com/company/about-mattermost/mindsets#brown-m-and-ms) that may require either a future interviewer to ask about or the Hiring Manager to cover during a reference check.
```

> ### Interview Project

Reviewer: ```suggestion
### Interview project
```

> Candidates may be asked to complete a project as part of the interview process. Hiring Managers will indicate this need during the in-take. Hiring Managers can create their own project. Reach out to Recruiting for sample projects. 

Reviewer: ```suggestion
Candidates may be asked to complete a project as part of the interview process. Hiring Managers will indicate this need during the intake. Hiring Managers can create their own project. Reach out to Recruiting for sample projects. 
```
